packages/forma-36-react-components/package.json

```diff
@@ -31,10 +31,13 @@
     "@storybook/addon-notes": "^5.2.1",
     "@storybook/addon-storysource": "^5.2.1",
     "@storybook/react": "^5.2.1",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.4.0",
     "@types/classnames": "2.2.7",
     "@types/enzyme": "3.9.1",
-    "@types/jest": "24.0.11",
-    "@types/jest-axe": "2.2.4",
+    "@types/is-hotkey": "^0.1.1",
```

Comment:
`@types` version bump is necessary because `@testing-library/jest-dom` included newer `jest` type augmentations.